react-components/components.md

```diff
@@ -0,0 +1,396 @@
+# Components
+
+## Introduction
+
+Sofia is a teacher, and she's creating a digital attendance webapp to help her at work. She wants to list all the students in her class.
+
+She knows that React will help her create the UI of her attendance webapp. But how does she turn her idea into React components?
+
+This lesson will follow the story of Sofia, who will learn the syntax to create and use two components.
+
+## Our Current Definition of Component
+
+**Components** in React are units of UI.
+
+All components can be rendered to the page, hold data, manage their own state, handle events, and contain other elements and components.
+
+Our understanding and definition of what a React component is and can do will grow over time. For now, let's learn the following.
```

Comment:
units of UI sounds uncomfortably vague. can we use the more familiar building block metaphor?
remember would be a better fit than learn here, but I like focus on for the more prolonged feeling of concentration it has.
```suggestion
**Components** in React are the building blocks of our UI.

All components can be rendered to the page, hold data, manage their own state, handle events, and contain other elements and components.

Our understanding and definition of what a React component is and can do will grow over time. For now, let's focus on the following core ideas.
```